app/src/main/kotlin/com/hippo/ehviewer/ui/scene/GalleryCommentsFragment.kt

```diff
@@ -258,9 +258,10 @@ fun GalleryCommentsScreen(galleryDetail: GalleryDetail, navigator: NavController
         },
     ) { paddingValues ->
         val keylineMargin = dimensionResource(id = R.dimen.keyline_margin)
+        var editTextMeasured by remember { mutableStateOf(80.dp) }
         Box(modifier = Modifier.fillMaxSize().imePadding()) {
             val additionalPadding = if (commenting) {
-                EditTextHeight
+                editTextMeasured
```

Comment:
Double navigation bar padding for comment list

Reply:
Done